org-timeline.el

```diff
@@ -147,25 +188,55 @@ Return new copy of STRING."
                           current-time))
                    (face (org-timeline--get-face)))
               (when (>= beg start-offset)
-                (push (list beg end face) tasks)))))))
-    (setq tasks (nreverse tasks))
+                (push (list beg end face txt line) tasks)))))))
```

Comment:
In some future pull request we should probably change the list to some actual data structure, since it is now growing pretty beg and it might become difficult to remember what data is at which position.  A `plist` or a `defstruct` would be a good choice.

Reply:
I'll do that in the next pull request (there isn't any feature left on the todo list, though).